.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature request
```

Comment:
Q There's `Ideas` in GitHub discussions. Do we still want to manage feature requests in issues? 
https://github.com/trinodb/trino-python-client/discussions/categories/ideas

Reply:
I think it makes sense to apply formal issue management to feature requests.

Reply:
The problem I think Yuya is referring to is that not feature requests are going to be accepted. So we should have some way to filter out what we'll actually work on and what we won't.

Since there's no dedicated backlog other than GitHub issues it makes sense to keep it clean from things that might be asked for but we won't implement (for whatever reasons).

Reply:
I'm personally fine with either direction since it seems like premature optimization to me.